### ISSUE TITLE
Can't select text while running the TUI

### ISSUE THREAD

**Issue by Pryz** [stealthrocket/dispatch#28]
When using the TUI `dispatch run -- ...` the user can't select copy/past anything in the terminal. This makes it difficult to share information while working locally.

Imagine a developer wanted to select an ID to query and dig further in the UI.

**Comment by achille-roussel** [comment]
This is due to capturing the mouse motion, we need to let the terminal handle the mouse to allow selection for copy/paste:

```diff
diff --git a/cli/run.go b/cli/run.go
index 5bd2665..af115f6 100644
--- a/cli/run.go
+++ b/cli/run.go
@@ -200,8 +200,7 @@ Run 'dispatch help run' to learn about Dispatch sessions.`, BridgeSession)
                                p := tea.NewProgram(tui,
                                        tea.WithContext(ctx),
                                        tea.WithoutSignalHandler(),
-                                       tea.WithoutCatchPanics(),
-                                       tea.WithMouseCellMotion())
+                                       tea.WithoutCatchPanics())
                                wg.Add(1)
                                go func() {
                                        defer wg.Done()
```

☝️ applying this diff addressed the issue.

**Comment by chriso** [comment]
This should be fixed on the #25 branch.
